pylintrc

```diff
@@ -1,10 +1,15 @@
+[MASTER]
+ignore=tests
```

Comment:
Do we really want to ignore test in linting? In my opinion we have to keep them readable too.

Reply:
Ok, removing the ignore.